parse_patent.py

```diff
@@ -42,22 +43,22 @@ def parse_uspto_file(bs, logging=False):
         for el in classes.find_all('classification-ipcr'):
 
             section = el.find('section').text
-                        
+
             classification  = section
             classification += el.find('class').text
             classification += el.find('subclass').text
-            
+
             group = el.find('main-group').text + "/"
             group += el.find('subgroup').text
 
             sections[section] = True
             section_classes[section+el.find('class').text] = True
             section_class_subclasses[classification] = True
             section_class_subclass_groups[classification+" "+group] = True
-            
+
     authors = []
-    for parties in bs.find_all('parties'):
-        for applicants in parties.find_all('applicants'):
+    for parties in bs.find_all(re.compile('^.*parties')):
+        for applicants in parties.find_all(re.compile('inventors')):
```

Comment:
author fix

Reply:
We probably want to know the entities as well

For instance, I'd like to know all patents from organization X